react-app/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
do we need this for a minimal example?

Reply:
Used CRA for this setup to save on time. Drawback is that we get some stuff we probably don't need. We could remove it, but it's not a big deal that it's here either.